oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCUserManager.java

```diff
@@ -234,7 +237,21 @@ private int sendLogout(Endpoint logoutEndpoint, IDTokenClaimsSet idToken) throws
 
     private void checkAllowedGroups(UserInfo userInfo) throws OIDCException
     {
-        List<String> providerGroups = getClaim(this.configuration.getGroupClaim(), userInfo);
+    	List<String> providerGroups = null;
+        Object providerGroupsObj = getClaim(this.configuration.getGroupClaim(), userInfo);
+        if (this.configuration.isGroupCommaDelimited()) {        
```

Comment:
I agree on the configurability of separator, but don't you think that a list of values, instead of a single value, could carry potentially unwanted split?